modules/configuration-ovnk-multi-network-policy.adoc

```diff
@@ -26,6 +26,8 @@ a|
 
 |====
 
+You can use the `k8s.v1.cni.cncf.io/policy-for` annotation in a `MultiNetworkPolicy` CRD to specify what `NetworkAttachmentDefinition` (NAD) from a pod with multiple networks applies to a secondary network interface.
```

Comment:
I don't know if it's just me, but I find this sentence confusing. Keep in mind I'm not a native speaker so. 

I think the important part is to indicate that annotation is used to specify the **network** to which the policy will be applied. And that network is defined by pointing the policy to an existent NAD. 

Can you somehow simplify the sentence to convey that information ? I've tried to but failed.